flake.nix

```diff
@@ -0,0 +1,34 @@
+{
+  description = "ngrok docs";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
+
+    flake-utils = {
+      url = "github:numtide/flake-utils";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+        };
+        node-toolchain = with pkgs; [
+          nodejs
+          nodePackages.pnpm
+        ];
+      in
+      {
+        devShell = pkgs.mkShell {
+          shellHook = ''
+            export PATH="$PATH:node_modules/.bin"
```

Comment:
:grimacing: 

relative paths in the `PATH` variable are a security hazard. Can we `git rev-parse --show-toplevel` to pin it to the repo root?